interface.html

```diff
@@ -239,7 +237,7 @@ <h3>Submission</h3>
                 </div>
               </div>
             </div>
-            <div class="form-group">
+            <div  v-if="!settings.dataStore.length" class="form-group">
```

Comment:
have you confirmed dataStore is always an array and you don't need an explicit check to avoid this line failing?

Reply:
datastore is either an empty array or an array with values